runtime/genesis/proto/genesis.proto

```diff
@@ -21,18 +21,18 @@ message GenesisState {
   Params params = 10;
 }
 
-// DISCUSS(drewskey): Explore a more general purpose "feature flag" like approach for this.
+// TECHDEBT: Explore a more general purpose "feature flag" approach that makes it easy to add/remove
+// parametesr and add activation heights for them as well.
 message Params {
+  reserved 4, 59; // Deprecated
```

Comment:
TOL: I agree that this is the right approach but since we are "early" perhaps we could just simply change the field indexes instead of reserving the ones that we removed.

It would make, IMHO, things easier to understand to newcomers expecially because there's no history yet around this fields. 
Once we are live/in testnet, of course we should deprecate them this way.